lib/oli/delivery/attempts/page_lifecycle/hierarchy.ex

```diff
@@ -83,7 +83,8 @@ defmodule Oli.Delivery.Attempts.PageLifecycle.Hierarchy do
       |> DateTime.truncate(:second)
 
     # Create the activity attempts, in bulk
-    Enum.map(activity_revisions, fn r ->
+    Enum.filter(activity_revisions, fn r -> !is_nil(r) end)
```

Comment:
I think piping activity_revisions into the filter would make this easier to grok.